registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/SyncRegistrationServiceImpl.java

```diff
@@ -217,22 +217,24 @@ public SyncRegistrationServiceImpl() {
 		boolean isTransactionSuccessful = false;
 		try {
 			for (SyncRegistrationDto registrationDto : resgistrationDtos) {
-				if(registrationDto.getAdditionalInfoReqId()!=null && registrationDto.getPacketId()!=null
-					&& !registrationDto.getAdditionalInfoReqId().isBlank() && !registrationDto.getPacketId().isBlank()
-					&& validateAdditionalInformation(registrationDto, syncResponseList)) {
+				if(registrationDto.getPacketId()!=null && !registrationDto.getPacketId().isBlank()){
+					if(registrationDto.getAdditionalInfoReqId()!=null && !registrationDto.getAdditionalInfoReqId().isBlank()
+							&& !registrationDto.getAdditionalInfoReqId().contains(registrationDto.getRegistrationType())) {
```

Comment:
Please remove this -
if(registrationDto.getAdditionalInfoReqId()!=null && !registrationDto.getAdditionalInfoReqId().isBlank()
							&& !registrationDto.getAdditionalInfoReqId().contains(registrationDto.getRegistrationType())) {